docs/source/basic_usage.rst

```diff
@@ -254,3 +254,20 @@ interact with it. Even better, you can **go back in time** because you know what
     vdb <your_json_report>
 
 For detailed commands, please refer to :doc:`virtual_debug`
+
+Compress Your Report
+--------------------
+
+VizTracer supports compressing your json report. The general compress ratio is about 1% - 3% for a large report.
```

Comment:
I think the compression ratio is defined as `uncompressed size / compressed size` [wiki](https://en.wikipedia.org/wiki/Data_compression_ratio). Here we could say 50:1 to 100:1 maybe.